src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -580,7 +623,11 @@ export class QuickBar extends LitElement {
 
   private _filterItems = memoizeOne(
     (items: QuickBarItem[], filter: string): QuickBarItem[] => {
-      return fuzzyFilterSort<QuickBarItem>(filter.trimLeft(), items);
+      return fuzzyFilterSort<QuickBarItem>(
+        filter.trimLeft(),
+        items,
+        createMatchDecorator("<span class='highlight-letter'>", "</span>")
```

Comment:
Or maybe even better:
```suggestion
        createMatchDecorator(html`<span class='highlight-letter'>${letter}</span>`)
```

Reply:
And `lit-html` to render the items